src/drivers/gitlab.js

```diff
@@ -69,6 +70,9 @@ class Gitlab {
   async commitCommentCreate(opts = {}) {
     const { commitSha, report } = opts;
 
+    if (report.length >= MAX_COMMENT_SIZE)
+      throw new Error('GitLab Comment too Large');
```

Comment:
Also better error message should suggest cause & solutions:

- split into multiple reports
- publish fewer images
- don't use `--publish-native` as it encodes images in `base64`, easily hitting the limit
- what is the max char limit (1M)